// angular2-modal/index.js
"use strict";
function __export(m) {
    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
}
__export(require('./framework/fluent-assign'));
var createComponent_1 = require('./framework/createComponent');
exports.createComponent = createComponent_1.createComponent;
var dialog_ref_1 = require('./models/dialog-ref');
exports.DialogRef = dialog_ref_1.DialogRef;
var tokens_1 = require('./models/tokens');
exports.DROP_IN_TYPE = tokens_1.DROP_IN_TYPE;
exports.ModalDropInFactory = tokens_1.ModalDropInFactory;
exports.ModalBackdropComponent = tokens_1.ModalBackdropComponent;
exports.ModalCompileConfig = tokens_1.ModalCompileConfig;
exports.ModalRenderer = tokens_1.ModalRenderer;
var providers_1 = require('./providers');
exports.Modal = providers_1.Modal;
exports.DOMModalRenderer = providers_1.DOMModalRenderer;
var modal_context_1 = require('./models/modal-context');
exports.DEFAULT_VALUES = modal_context_1.DEFAULT_VALUES;
exports.ModalContext = modal_context_1.ModalContext;
exports.ModalContextBuilder = modal_context_1.ModalContextBuilder;
var modal_open_context_1 = require('./models/modal-open-context');
exports.ModalOpenContext = modal_open_context_1.ModalOpenContext;
exports.ModalOpenContextBuilder = modal_open_context_1.ModalOpenContextBuilder;
var angular2_modal_module_1 = require('./angular2-modal.module');
exports.ModalModule = angular2_modal_module_1.ModalModule;
